md/single-sign-on-with-kerberos.md

```diff
@@ -0,0 +1,331 @@
+			default_tgs_enctypes = aes256-cts-hmac-sha1-96 aes128-cts rc4-hmac des3-cbc-sha1 des-cbc-md5 des-cbc-crc
+			permitted_enctypes   = aes256-cts-hmac-sha1-96 aes128-cts rc4-hmac des3-cbc-sha1 des-cbc-md5 des-cbc-crc
+
+		[realms]
+	-->		BONITA.LOCAL  = {
+	-->			kdc = DC.bonita.local 
+	-->			default_domain = BONITA.LOCAL 
+			}
+
+		[domain_realm]
+	-->		.BONITA.LOCAL = BONITA.LOCAL
+
+```
+	
+if you want to use the AES256-CTS encryption type, you need to update the Java security libraries (Java Cryptography Extension (JCE) Unlimited Strength) to those for Strong Encryption. Depending on your java version, you might have to download some extra files or not.
+
+	* For Java updates > Java 8 u162 and java 9, the unlimited policy is enabled by default. You no longer need to install the policy file in the JRE or set the security property crypto.policy
+	* For Java updates < Java 8 u162, you have to download the security libraries [Here](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html)
+		These libraries need to be put in jre/lib/security and jdk/jre/lib/security.
+
+3. In the following folder `<BUNDLE_HOME>/server/conf`,
+	edit the login.conf file as follows:
+	
+```	
+	spnego-client {
+		com.sun.security.auth.module.Krb5LoginModule required;
+	};
+
+	spnego-server {
+		com.sun.security.auth.module.Krb5LoginModule required
+		storeKey=true
+		isInitiator=false;
+	};
+```
+	
+4. In the tenant_portal folder of each existing tenant: `<BUNDLE_HOME>/setup/platform_conf/current/tenants/<TENANT_ID>/tenant_portal`,
+   edit the authenticationManager-config.properties as follows:
+   
+```
+		# saml.logout.global = false
+		# auth.tenant.admin.username = install
+		# auth.passphrase = BonitaBPM
+   
+	-->	auth.AuthenticationManager = org.bonitasoft.console.common.server.auth.impl.kerberos.RemoteAuthenticationManagerImpl
+	-->	kerberos.filter.active = true
+	-->	kerberos.auth.standard.allowed = true
+	-->	auth.tenant.admin.username = install
+	-->	auth.tenant.standard.whitelist = william.jobs
+	-->	auth.passphrase = Bonita
+
+		# auth.AuthenticationManager = org.bonitasoft.console.common.server.auth.impl.oauth.OAuthAuthenticationManagerImpl
+		# OAuth.serviceProvider = LinkedIn
+		# OAuth.consumerKey = ove2vcdjptar
+   -->  logout.link.hidden=true 
+```
+
+Make sure to [set the right tenant admin username](multi-tenancy-and-tenant-configuration#toc2).
+It is recommended to also replace the value of the passphrase (property auth.passphrase) which is used by the engine to verify the authentication request.
+The value must be the same as in the file **bonita-tenant-sp-custom.properties**.  
+If some users need to bypass kerberos authentication method, you can authorize it by setting the property `kerberos.auth.standard.allowed` to true. Users will then be able to log in using the portal login page (/login.jsp) provided they have a bonita account and their password is different from their username.
```

Comment:
I think we should remove the "some" (If some users need to bypass) and add a sentence:
If only a limited group of users need to bypass kerberos authentication method you can restrain it by setting the property `kerberos.auth.standard.allowed` to false and setting the property `auth.tenant.standard.whitelist` with the list of authorized usernames (coma separated).